cleverpush/src/main/java/com/cleverpush/banner/AppBannerCarouselAdapter.java

```diff
@@ -537,6 +537,14 @@ private void composeHtmlBanner(LinearLayout body, String htmlContent) {
         webView.addJavascriptInterface(new CleverpushInterface(), "CleverPush");
         webView.setWebViewClient(new AppBannerWebViewClient());
 
+        // Ensure WebView is scrollable
+        webView.setOnTouchListener((v, event) -> {
+          if (event.getAction() == MotionEvent.ACTION_DOWN || event.getAction() == MotionEvent.ACTION_MOVE) {
+            v.getParent().requestDisallowInterceptTouchEvent(true);
+          }
```

Comment:
no need for requestDisallowInterceptTouchEvent(false); ?

Reply:
@unnaticleverpush You can merge + Release yourself if this is sorted.

Reply:
@cpKevinrodriguesnobre there is no need to re-enable parent intercepting because we are only disabling it during specific touch events (scrolling or initial touch down). Once the touch event ends (ACTION_UP), the default behavior will resume without explicitly needing to call requestDisallowInterceptTouchEvent(false).